cg_hermes/constants/tags.py

```diff
@@ -6,6 +6,7 @@ class AlignmentTags(StrEnum):
     BAM: str = "bam"
     BAM_INDEX: str = "bam-index"
     BAM_MT: str = "bam-mt"
+    BAM_MT_INDEX: str = "bam-mt-index"
```

Comment:
could instead be two tags `bam-mt` and `index`?

Reply:
There is not yet a "index" tag. So I think it is too early to implement in this PR, it needs to be done as a separate project